test/metabase/pulse/api/pulse_test.clj

```diff
@@ -991,6 +991,45 @@
                         :recipient-type nil}
                        (mt/summarize-multipart-single-email (-> channel-messages :channel/email first) #"Daily Sad Toucans")))))))))))
 
+(deftest send-test-pulse-to-non-user-test
+  (testing "POST /api/pulse/test"
```

Comment:
we should include some testing context here. I'd put a description saying: sending test email to non user won't include unsubscribe link. and also the github issue `#43391`

Reply:
Thanks! I've missed to update the description, you're absolutely correct.

Reply:
it's always easier to debug a small test than a big one :)